docs/advanced-topics/verification.rst

```diff
@@ -0,0 +1,126 @@
+    tab with the label: “TRS CWL Path” or “TRS WDL Path”
+5.  Provide the version_id of the tool/workflow to verify. It can be any
+    version listed in the Version tab of the tool/workflow.
+    `dockstore-tool-md5sum <https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:master?tab=versions>`__
+    has the following versions currently: 1.0.4, master, develop, 1.0.3,
+    latest, 1.0.2, 1.0.1, and 1.0.0.
+6.  Provide the “relative_path” of the test parameter file being
+    verified. The path of the test parameter file is relative to the
+    primary descriptor. This path can be found using the `files
+    endpoint`_ or by viewing the files tab of a tool/workflow such as:
+    https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4?tab=files
+    and then further selecting the Test Parameter Files tab and view the
+    right-most “File” dropdown. This relative path must be a test
+    parameter file, providing a descriptor will not work.
+7.  Provide the “platform”. Some examples are: HCA, Cromwell, Arvados,
+    etc
+8.  Select the “verified” status either as “true” or “false”. Use “true”
+    to verify, “false” to “unverify”.
+9.  Provide “metadata”, this is typically the verifier’s identity which
+    can be something like “GA4GH/DREAM Challenge”
+10. Lastly, provide your Dockstore token using the lock icon at the top
+    right of the endpoint
+
+Below is a screenshot of someone verifying the “test.json” test
+parameter file of the “master” version of the “dockstore-tool-md5sum”
+tool.
+
+.. figure:: /assets/images/docs/verification/swagger.png
+   :alt: Swagger-UI
+
+   Swagger-UI
+
+The curl command results in something like:
+
+::
+
+   curl -X POST "https://dockstore.org/api/api/ga4gh/v2/extended/quay.io%2Fbriandoconnor%2Fdockstore-tool-md5sum/versions/master/CWL/tests/test.json?platform=Dockstore%20CLI&verified=true&metadata=Phase%201%20GA4GH%20Tool%20Execution%20Challenge" -H  "accept: application/json" -H  "Authorization: Bearer iamafakebearertoken"
+
+A successful response will result in something like:
+
+::
+
+   {
+     "Dockstore CLI": {
+       "metadata": "Phase 1 GA4GH Tool Execution Challenge",
+       "verified": true
+     }
+   }
+
+Viewing on Dockstore.org
+========================
+
+There are 3 new indicators in Dockstore.org that indicates whether or
+not the tool/workflow is correctly verified.
+
+To see if the verified tool/workflow is correctly displaying to others
+on Dockstore.org, first go to the page of the tool/workflow such as
+https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4?tab=info.
+Since this tool/workflow is correctly verified, the 3 indicators can be
+seen:
+
+.. figure:: /assets/images/docs/verification/tool.png
+   :alt: Tool Page
+
+   Tool Page
+
+1. At the top left, the checkmark indicates that at least one of the
+   tool/workflow’s version has been verified. In general, this
```

Comment:
Why "In general"? Are there cases where it's displayed, but the tool/workflow is not verified?

Reply:
Because all the other tool/workflow versions may not be verified.  

It's worded that way so people don't think that every single version of a verified tool/workflow is verified.